CONTRIBUTING.md

```diff
@@ -9,6 +9,7 @@ This document serves as a general guide for contributing to Ruffle. Follow your
 * [Getting Started](#getting-started)
 * [Ways to Contribute](#ways-to-contribute)
     * [Test your favorite Flash content](#test-your-favorite-flash-content)
+    * [Translate Ruffle to your language](#user-content-translate-ruffle-to-your-language)
```

Comment:
Nit: Is the `user-content-` part added on purpose?

Reply:
I noticed it in as the id in the source code, but it looks like it's not needed.